.packit.yaml

```diff
@@ -77,3 +77,20 @@ jobs:
     packages: [python-podman-fedora]
     dist_git_branches:
       - fedora-branched # rawhide updates are created automatically
+
+  # Test linting on the codebase
+  # This test might break based on the OS and lint used, so we follow fedora-latest as a reference
+  - job: tests
+    trigger: pull_request
+    targets:
+      - fedora-latest-stable
+    skip_build: true
+
+  # Test integration on all the targets
+  - job: tests
+    trigger: pull_request
+    targets:
+      - fedora-all
+      - centos-stream-9
+      - centos-stream-10
+      - epel-9
```

Comment:
It might be good to have separate jobs for fedora and centos even if the tests are the same. And each separate job can use the `packages: ..` key value as mentioned in the previous comment.

FWIW, we're tending towards avoiding epel (rhel) targets on other projects as things often break / fail to build because of the slow-moving nature. But if it works here and you want to keep it, fine by me. Might wanna have epel targets tested separately using the `python-podman-rhel` key and keep it separate from centos.

Reply:
ok I am fixing the .packit file, thanks